package.json

```diff
@@ -10,16 +10,16 @@
     "react-scripts": "3.2.0"
   },
   "scripts": {
+    "test" : "eslint *.js src/*",
     "start": "rescripts start",
     "build": "rescripts build",
-    "test": "rescripts test",
```

Comment:
My bad, i think its best you combine both of these commands: `eslint *.js src/* && rescripts test`

Reply:
combining